x/evm/types/types.go

```diff
@@ -72,15 +72,15 @@ const (
 		}
 	]`
 	AxelarGatewayCommandMintToken            = "mintToken"
-	mintTokenMaxGasCost                      = 200000
+	mintTokenMaxGasCost                      = 150000
 	AxelarGatewayCommandDeployToken          = "deployToken"
-	deployTokenMaxGasCost                    = 1500000
+	deployTokenMaxGasCost                    = 1400000
 	AxelarGatewayCommandBurnToken            = "burnToken"
-	burnTokenMaxGasCost                      = 200000
+	burnTokenMaxGasCost                      = 400000
```

Comment:
Why is burn so much more expensive now?

Reply:
external token

Reply:
I see, can we be smarter about it then? Have two constants, one for external erc20 burn and one for our wrapped ones. And depending on the command, the appropriate one is added. We're filling up our batches fairly often already, and by doubling the burn cost for our wrapped tokens too, we'll be much less efficient. Avalanche, where we have most throughput rn, has had pretty high variable gas costs recently ($10-50 per batch).

Reply:
Yes but I'd rather save that for later and fix it separately. External tokens also affect other commands like deployToken and mintToken.